script.js

```diff
@@ -289,13 +289,14 @@ function appendStory(story, board, storyUrl, statusBadge, estimate) {
   let storyType = story.story_type;
   let email = getEmailAddress(story.owned_by_id);
   let urlForGravatar = gravatar(email, 44);
```

Comment:
You need to format the options as an object. Check what it's expecting in your `gravatar` function:

```
  //set some defaults, just in case
  options = {
    size: options.size || "50",
    rating: options.rating || "g",
    secure: options.secure || location.protocol === "https:",
    backup: options.backup || ""
  };
```

So you'd want something like:

```
let urlForGravatar  = gravatar(email, { size: "25" });
```